user_docs/en/changes.t2t

```diff
@@ -4,21 +4,26 @@ What's New in NVDA
 %!includeconf: ../changes.t2tconf
 
 = 2021.2 =
+This release introduces preliminary Windows 11 support.
+While Windows 11 is yet to be released, this release has been tested on preview versions of Windows 11.
+This includes an important fix for Screen Curtain (see important notes).
+The COM Registration Fixing Tool can now resolve more problems when running NVDA.
+There are updates to the synthesizer eSpeak and braille translator LibLouis.
+There are also various bug fixes and improvements, notably for braille support and Windows terminals, calculator, emoji panel and clipboard history.
+
 
 == Important notes ==
-- Screen Curtain on previous releases of NVDA is no longer compatible with versions of Windows newer than Windows 10 21H2 (10.0.19044).
+Screen Curtain is no longer compatible when using versions of NVDA older than 2021.2 with Windows 10 21H2 (10.0.19044) or later.

```

Comment:
I wonder if there is another way to word this. I think it could be misinterpreted as Screen Curtain was MADE incompatible, rather than newest versions of Windows have changed behavior breaking the screen curtain feature in older releases of NVDA.

What about:
```suggestion
Please update to this release if you rely on Screen Curtain and use Windows 10 21H2 (10.0.19044) or later.
 - The screen curtain feature had to be updated to support the newest versions of Windows.
```